README.rst

```diff
@@ -91,6 +91,8 @@ The defintion for each components is:
 - **version**: the version of the package. Optional.
 - **qualifiers**: extra qualifying data for a package such as an OS,
   architecture, a distro, etc. Optional and type-specific.
+  Multiple qualifiers must be separated with "&" and the keys should be
+  sorted by key in the "C" locale.
```

Comment:
BTW, just a side note for the wording for this new PR: `sorted by key in the "C" locale.` should be clarified and/or simplified. Also I am not sure this needs to be made so prominent as we do not provide so much details at that level for other components. 
Note that the purl is already specified as being ASCII-only and the qualifier sorting is stated as `sort this list of qualifier strings lexicographically` which would be clear with ASCII and unambiguous as the keys are specified as being lowercase only, and there cannot be two qualifiers with the same key.   This boils down to sorting digits first in ascending numeric order then letters in alphabetic order which is alright the C locale, but is also quite simply the default sort order in Python, Ruby, JavaScript. If we introduce the "C" locale there is a bit more complexity to process. 

But to your point making the syntax and sorting should be made more prominent, likely under the "Rules for each purl component/qualifiers"

Reply:
"Sorted by key lexicographically" is better than being unstated. I was trying for maximum precision. As long as there are only ASCII digits and lowercase letters I *think* locale doesn't matter, though there may be an exception.

Reply:
I think we are good with lexicographic sorting (being a fancy word for alphabetical/alphanumeric) as we have only ASCII as specified. This was not raised as an issue since so we are good this way. I found that invoking a "C" locale has been raising more questions and head scratching that help